CRM/Upgrade/Incremental/sql/5.63.alpha1.mysql.tpl

```diff
@@ -12,3 +12,6 @@ INSERT IGNORE INTO civicrm_navigation
 VALUES
     ( {$domainID}, 'https://docs.civicrm.org/user/?src=iam', '{ts escape="sql" skip="true"}User Guide{/ts}', 'User Guide', NULL, 'AND', @adminHelplastID, '1', NULL, 1 ),
     ( {$domainID}, 'https://civicrm.org/help?src=iam',       '{ts escape="sql" skip="true"}Get Help{/ts}',   'Get Help',   NULL, 'AND', @adminHelplastID, '1', NULL, 2 );
+
+{* https://github.com/civicrm/civicrm-core/pull/26145 *}
+UPDATE civicrm_uf_field SET label = '{ts escape="sql"}Contribution Date{/ts}' WHERE field_name = 'receive_date';
```

Comment:
The test is failing because this would need to be done in php and accounting for multilingual where the field is called label_enUS, etc, like in FiveThirtyOne.php. But I think it would also be ok to leave it out so we can get this merged.

Reply:
```suggestion
{if $multilingual}
  {foreach from=$locales item=locale}
      UPDATE IGNORE civicrm_uf_field SET label_{$locale} = '{ts escape="sql"}Contribution Date{/ts}' WHERE field_name = 'receive_date';
  {/foreach}
{else}
UPDATE IGNORE civicrm_uf_field SET label = '{ts escape="sql"}Contribution Date{/ts}' WHERE field_name = 'receive_date';
{/if}
```
Sorry, I should have thought to mention that. This should do the job. It still works in SQL, @demeritcowboy?

Reply:
I mean, it does work, I tested it. But there's no specific reason we can't still use SQL here, is what I meant.

Reply:
@larssandergreen, thanks for the suggestion it looks good, is it okay to commit?
 or do we want to leave it as it is?

Reply:
It can't be left as is, and now there is also a merge conflict. Either sql or php but also it should only change it if the current label is Date Received, otherwise it will overwrite someone's preferred wording if they already changed it in the profile.

Reply:
Hi @olayiwola-compucorp it's still failing tests since this is the same code as before. Can I suggest leaving this out and then we can merge the rest? You can always put up another PR after.